deploy_managed_infra/managed_infra_template.json

```diff
@@ -99,92 +99,92 @@
                     {
                         "record": "average_dcgm_gpu_temp",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_gpu_temp) by (instance, subscription, cluster, job_id)"
                     },
                     {
                         "record": "max_dcgm_gpu_temp",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_gpu_temp) by (instance, subscription, cluster, job_id)"
```

Comment:
should this expression be max(dcgm_gpu_temp) ?

Reply:
addressed